File: java-swiss-back/src/main/java/com/back/back/entity/AlarmEntity.java
package com.back.back.entity;

import java.io.Serializable;
import java.time.format.DateTimeFormatter;
import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name="alarm")
public class AlarmEntity implements Serializable {

    private static final long serialVersionUID = 1L;
    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");

    @Id
    @GeneratedValue
    private Long id;
    private String city;
    private String country;
    private double threshold;
    private boolean status;
    @Column(name="created_on")
    private LocalDateTime createdOn;
    @Column(name="updated_on")
    private LocalDateTime updatedOn;
    @Column(name="audio_path")
    private String path;
    private String classe;
    @Column(name="model_type")
    private String modelType;

    public boolean getStatus() {
        return this.status;
    }

    public String getUpdatedOn() {
        return this.updatedOn.format(formatter);
    }

    public void setUpdatedOn(final String updatedOn) {
        this.updatedOn = LocalDateTime.parse(updatedOn, formatter);
    }

    public String getCreatedOn() {
        return this.createdOn.format(formatter);
    }

    public void setCreatedOn(final String createdOn) {
        this.createdOn = LocalDateTime.parse(createdOn, formatter);
    }

}

File: inference/entity/AlarmEntity.py
from datetime import datetime

class AlarmEntity:
    def __init__(self, id: int = 0,
                       city: str = "Rennes",
                       country: str = "FR",
                       threshold: float = 0.80,
                       status: bool = True,
                       created_on = datetime.now(),
                       updated_on = datetime.now(),
                       audio_path: str = "",
                       classe: str = "Unknown",
                       model_type: str = "") -> None:
        self.id = id
        self.city = city
        self.country = country
        self.threshold = threshold
        self.status = status
        self.created_on = created_on
        self.updated_on = updated_on
        self.audio_path = audio_path
        self.classe = classe
        self.model_type = model_type

    def __repr__(self) -> str:
        return f"""*** Entity: {self.id} ***
                               {self.city}\n
                               {self.country}\n
                               {self.threshold}\n
                               {self.status}\n
                               {self.created_on}\n
                               {self.updated_on}\n
                               {self.audio_path}\n
                               {self.classe}\n"""